core/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/telemetry/CloudVendor.scala

```diff
@@ -0,0 +1,13 @@
+package com.snowplowanalytics.snowplow.collectors.scalastream.telemetry
+
+sealed trait CloudVendor {
+  override def toString: String = this match {
+    case CloudVendor.Aws => "AWS"
+    case CloudVendor.Gcp => "GCP"
+  }
```

Comment:
We probably want to use an encoder here and not use `toString`. See [here](https://github.com/snowplow/enrich/blob/feature/file-sink-rotate/modules/common-fs2/src/main/scala/com/snowplowanalytics/snowplow/enrich/common/fs2/Telemetry.scala#L121)

Reply:
The Collector does not use any typeclasses. The static method fits the style of the project.

Reply:
Core makes use of the ProjInfo to print the help message. 

https://github.com/voropaevp/stream-collector/blob/9cf5921e21bfdb45b2cb72e2736924c4ef2fb3ea/core/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/Collector.scala#L45

We could pass this into the parseConfig from the `Main` modules. 
` val (collectorConf, akkaConf) = parseConfig(args, buildInfo....)`

Do you have alternative suggestions?

Reply:
I'm confused, this comment has nothing to do with the encoder right ?

Reply:
> We could pass this into the parseConfig from the Main modules

Yes I think that each `main()` should pass the `BuildInfo` to the core. That's what we do in enrich